src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testPieces/DeliveryPipeTest.java

```diff
@@ -8,5 +8,19 @@
 
     @Test
     public void getDeliveryWarmWater() {
+        DeliveryPipe deliveryPipe = new DeliveryPipe();
+        boolean expected = true;
+        boolean actual = deliveryPipe.getDeliveryWarmWater();
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setDeliveryWarmWater() {
```

Comment:
the PR has to follow our convention. you need to change the name of method

Reply:
Okay thanks for the observation

Reply:
Already done